src/homepageExperience/components/steps/CreateToken.tsx

```diff
@@ -18,19 +18,15 @@ import {event} from 'src/cloud/utils/reporting'
 
 type ReduxProps = ConnectedProps<typeof connector>
 
-// Events log handling
-const logCopyCodeSnippet = () => {
-  event('firstMile.pythonWizard.createToken.code.copied')
+type OwnProps = {
+  wizardEventName: string
 }
 
-const logDocsOpened = () => {
-  event('firstMile.pythonWizard.createToken.docs.opened')
-}
-
-const CreateTokenComponent: FC<ReduxProps & RouteComponentProps> = ({
+const CreateTokenComponent: FC<ReduxProps & RouteComponentProps & OwnProps> = ({
   showOverlay,
   dismissOverlay,
   getAllResources,
+  wizardEventName,
 }) => {
   const org = useSelector(getOrg)
```

Comment:
`CreateToken` is a shared file, so needed the prop to pass in `wizardEventName`

Reply:
good catch! 🎣